packages/react-api/package.json

```diff
@@ -53,12 +53,10 @@
     "webpack": "^5.24.2",
     "webpack-cli": "^4.5.0"
   },
-  "dependencies": {
-    "@babel/runtime": "^7.13.9"
```

Comment:
@padawannn Is there a particular reason to remove here the runtime?

Reply:
ups